releases/adm/sciencebeam-alert-rules.yaml

```diff
@@ -0,0 +1,13 @@
+apiVersion: monitoring.coreos.com/v1
+kind: PrometheusRule
+metadata:
+  name: sciencebeam.rules
+  namespace: adm
+  labels:
+    app: sciencebeam
+spec:
+  groups:
+  - name: sciencebeam
+    rules:
+    - alert: ScienceBeamServerError
```

Comment:
HI @erkannt I believe this is the line it is complaining about:

```
component=sync-loop error="getting unlocked automated resources: scanning multidoc from \"releases/adm/sciencebeam-alert-rules.yaml\": yaml: line 12: did not find expected key"
```

Should `ScienceBeamServerError` be defined elsewhere? I thought this is the defining `ScienceBeamServerError`.